### DISCUSSION TITLE
[Art competition] Access Control

### DISCUSSION THREAD

**Discussion by Byunk** [prosyslab-classroom/cs348-information-security#410] [Art competition]
![access-control](https://github.com/prosyslab-classroom/cs348-information-security/assets/60650372/eeeb69c0-6889-4bf8-9607-d83fa08df93b)

## Short Description
For me, Access Control means trying on Cinderella's glass shoes.

## Long Description (up to 3 sentences)
Access Control resembles the process of a prince trying on a girl with Cinderella's shoes. The girl tries to trick the prince, praying that his feet fit in his shoes to marry him, and the prince thoroughly verifies with the key (shoes) to finding Cinderella.

**Comment by KihongHeo** [comment]
Good. Is this method sound or complete?
